hazelcast-jet-core/src/main/java/com/hazelcast/jet/DAG.java

```diff
@@ -229,7 +230,19 @@ public Vertex getVertex(String vertexName) {
 
     @Override
     public String toString() {
-        return "Vertices " + verticesByName + "\nEdges " + edges;
+        final StringBuilder b = new StringBuilder("dag\n");
+        for (Iterator<Vertex> it = iterator(); it.hasNext();) {
+            final Vertex v = it.next();
+            b.append("    .vertex(").append(v.getName());
+            if (v.getLocalParallelism() != -1) {
+                b.append(".localParallelism(").append(v.getLocalParallelism()).append(')');
+            }
```

Comment:
About the `.localParallelism` on its own line, i tried it, but it makes the log entry take more vertical space. I'm worried about vertical space even without this change.